slicedimage/backends/_caching.py

```diff
@@ -9,23 +12,23 @@ class CachingBackend(Backend):
     def __init__(self, cacheroot, authoritative_backend):
         self._cacheroot = cacheroot
         self._authoritative_backend = authoritative_backend
+        self.cache = Cache(cacheroot, size_limit=int(5e9))
```

Comment:
consider making 5e9 a constant in this module.

alternatively, you can do:

```
    def __init__(self, cacheroot, authoritative_backend, size_limit=int(5e9)):
        self._cacheroot = cacheroot
        self._authoritative_backend = authoritative_backend
        self.cache = Cache(cacheroot, size_limit=size_limit)
```

rationale: easier to find the constant.